install/kubernetes/helm/istio-remote/templates/endpoints.yaml

```diff
@@ -1,113 +0,0 @@
-apiVersion: v1
```

Comment:
If removing this, then how to let the local istio connect to remote istio?

BTW: I also created a PR to add more checking for this at https://github.com/istio/istio/pull/6393

Reply:
agreed, I don't understand how this works properly.  @Rigdon could you expand on how one deploys Istio in this setup?

Reply:
The proxy containers in the remote cluster will use the remote endpoints directly as this is written. We could add some `ExternalName` services so that they are referencing local service names but I couldn't think of any immediate value in that.

Reply:
I can go ahead and update the README with details if we think this is generally the right direction.

Reply:
The README.md should not exist - it was removed once and replaced with a link to istio.io.  No sense updating README.md, although istio.io will need any updates that are relevant to changes made in the charts.

Reply:
👍

Reply:
@sdake @Rigdon @gyliu513  -- does removing this mandate using auto-sidecar injection?  (or pretty much mandate it?)

With the selectorless service + endpoint setup on the remote, we're able to make the manual injection workflow use the exact same `kube-inject` output for apps as if they were deployed on the cluster with the Istio control-plane.  IMO, this is fairly nice.  

However, I think a similar workflow could still work with this change if the `istioctl kube-inject` is run against a remote cluster with the `--meshConfigMapName` (or maybe `--injectConfigMapName`)  option pointing to the `istio` configmap in this change.  I hope to get time to play with that scenario.

Reply:
Yeah I guess I was hoping (I haven't been using manual injection at all) that `istioctl kube-inject` would honor the meshConfig in the target cluster. If that's not the case then adding the `ExternalName` type service may be a better option.

Reply:
thanks all, I think this is good to remove this as long as we can make sure manual injection works. @sdake @tiswanso @Rigdon

Reply:
@Rigdon @tiswanso I think the outstanding question that needs answered prior to merging this PR is whether or not manual injection works with it.  All of the e2e tests for multicluster depend on manual injection, and while i'm not opposed to changing those to automatic injection, we don't want to break the test cases with this merge.

Further we do need manual injection to work - atleast for 1.0 and until automatic injection has less tread on the tires.

Cheers
-steve